engines/glk/advsys/vm.cpp

```diff
@@ -593,8 +593,8 @@ bool VM::getWord(Common::String &line) {
 		_words.push_back(iw);
 		return true;
 	} else {
-		Common::String msg = Common::String::format(_("I don't know the word \"%s\".\n"), iw._text.c_str());
-		print(msg);
+		Common::U32String msg = Common::U32String::format(_("I don't know the word \"%s\".\n"), iw._text.c_str());
+		print(msg.encode());
```

Comment:
should things like these not use translations? print would not support chars from other than ascii (I think). i will also mark some more places like these, which I have some doubt